Readme.md

```diff
@@ -88,9 +88,10 @@ app.use(convert(session(app)));
 ### Options
 
   The cookie name is controlled by the `key` option, which defaults
-  to "koa:sess". All other options are passed to `ctx.cookies.get()` and
-  `ctx.cookies.set()` allowing you to control security, domain, path,
-  and signing among other settings.
+  to "koa:sess". The cookie key is controlled by the `genid` option,
```

Comment:
no, `genid` is only work with external store, cookie store won't run into this logic. 

```js

# cookie store
koa:sess -> session vlaue
# external store
koa:sess -> genid() -> session value
```